package.json

```diff
@@ -63,16 +63,15 @@
     "resolve-pathname": "^3.0.0",
     "styled-components": "^4.1.3",
     "use-wallet": "^0.4.4",
-    "web3": "^1.2.2",
-    "web3-eth-abi": "^1.2.4",
-    "web3-utils": "^1.2.2"
+    "web3": "1.2.6",
```

Comment:
Very weird; even though I'm pinning this, `@aragon/wrapper`'s web3 version is still using `1.2.8`.

The tests are still failing, and I'm guessing this must be it...